modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -437,14 +284,10 @@ def _handle_triggers_within_batch(self, batch: list[tuple[int, int, int]], trigg
             }
             self._persist_pipeline_log()
 
-            num_samples_in_trigger = self.grpc.get_number_of_samples(self.pipeline_id, trigger_id)
-            if num_samples_in_trigger > 0:
-                self.status_bar.update(demo="Training")
-                self._run_training(trigger_id)  # Blocks until training is done.
-            else:
-                logger.info(f"Skipping training on empty trigger {trigger_id}]")
-
-            self.status_bar.update(demo="Handling triggers")
+            self._run_training(trigger_id)  # Blocks until training is done.
```

Comment:
https://github.com/eth-easl/modyn/commit/2366cb494372f2672ecd186906b9bffe7d3843c8 got lost during the merge

Reply:
fixed